contrib/pyln-client/pyln/client/plugin.py

```diff
@@ -250,6 +250,9 @@ def add_option(self, name, default, description, opt_type="string"):
                 "Name {} is already used by another option".format(name)
             )
 
+        if opt_type not in ["string", "int", "bool"]:
```

Comment:
[commit msg]
> not sure if this is totally necessary, should we jsut let clightning
enforce the input?

Might be a bit redundant but an advantage is that one can see its mistake just by trying its plugin with `python3 myplugin.py`..